concordium-std/src/prims.rs

```diff
@@ -477,4 +477,33 @@ mod host_dummy_functions {
     fn hash_keccak_256(_data: *const u8, _data_len: u32, _output: *mut u8) {
         unimplemented!("Dummy function! Not to be executed")
     }
+
+    #[no_mangle]
+    fn report_error(
```

Comment:
I am wondering why this was not needed before?

Reply:
I was wondering about that as well.